roles/migrationcontroller/templates/controller.yml.j2

```diff
@@ -71,6 +71,10 @@ spec:
           value: webhook-server-secret
         - name: MIGRATION_REGISTRY_IMAGE
           value: {{ migration_registry_image_fqin }}
+{% if mig_controller_enable_cache|bool is true %}
```

Comment:
Why not the configmap?

Reply:
I put it in the configmap before merge, thanks for the suggestion.